src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7974,6 +7974,19 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureAllowsRefStructConstraint" xml:space="preserve">
     <value>allows ref struct constraint</value>
   </data>
+  <!-- PROTOTYPE: Condense -->
+  <data name="ERR_CannotApplyOverloadResolutionPriorityToOverride" xml:space="preserve">
```

Comment:
>ERR_CannotApplyOverloadResolutionPriorityToOverride [](http://example.com/codeflow?start=14&length=51)

Consider collapsing the 3 error messages into 1 or 2, especially if we also want to report an error when the attribute is applied to an explicit interface implementation.

Reply:
I'm just going to add a 4th for explicit implementations. Sorry, I like my specific errors.